src/js/utils/Template.js

```diff
@@ -3,6 +3,18 @@
   var templates = {};
 
   ns.Template = {
+    /**
+     * Takes in a templateId and values to replace the keys found in the template
+     * Requires Node-Webkit environment for file system access
```

Comment:
This seems odd. Did you have to do something special to get it working? This should be handle by the package.json or something, right?